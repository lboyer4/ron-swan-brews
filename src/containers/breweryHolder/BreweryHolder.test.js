import React from 'react';
import { shallow } from 'enzyme';
import { BreweryHolder, mapStateToProps } from './index.js';


describe('BreweryHolder', () => {
	let wrapper;
	let mockBreweries;
	let mockQuotes;

	beforeEach(() => {
		let mockBreweries = [ 
			{
				brewery: 'brew1'
			},
			{
				brewery: 'brew2'
			}
		]

		let mockQuotes = [
			'quote one',
			'quote two'
		]

		wrapper = shallow(
			<BreweryHolder
				breweries = {mockBreweries}
				quotes = {mockQuotes}
			/>
		)
	})

	it('should match snapshots', () => {
		expect(wrapper).toMatchSnapshot();
	})
})