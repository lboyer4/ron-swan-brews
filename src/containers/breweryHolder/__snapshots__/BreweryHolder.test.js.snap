// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreweryHolder should match snapshots 1`] = `
<section
  className="card-holder"
>
  <h1
    className="quote"
  >
    "
    "
  </h1>
</section>
`;
